elastic/kibana
# Remove skipping task logic from Alerting plugin

Towards: https://github.com/elastic/kibana/issues/176585

This PR is a follow-on of https://github.com/elastic/kibana/pull/177244 and removes the task skipping logic from the Alerting plugin.

## To verify

Rules should run as expected and `prepare_to_run_duration_ms` should not be in the event-log metrics.
Below query can be used to check metrics.

```
GET /.kibana-event-log-*/_search
{
  "query": {
    "bool" : {
      "must" : [
        { "term" : { "event.action" : "execute" } },
        { "term" : { "event.provider" : "alerting" } }
      ]
    }
  },
  "size": 100
}
```
